src/main/java/fi/otavanopisto/kuntaapi/server/integrations/management/ManagementPageProvider.java

```diff
@@ -263,48 +262,46 @@ public AttachmentData getPageImageData(OrganizationId organizationId, PageId pag
     return Collections.emptyList();
   }
 
-  private List<Page> translatePages(List<fi.otavanopisto.mwp.client.model.Page> mwpPages) {
+  private List<Page> translatePages(List<fi.otavanopisto.mwp.client.model.Page> managementPages) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused private "translatePages" method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUnusedPrivateMethod)